src/portal/lib/farms.ts

```diff
@@ -216,7 +213,7 @@ export async function addNodePublicConfig(
   address: string,
   api: { tx: { tfgridModule: { addNodePublicConfig: (arg0: any, arg1: any, arg2: any) => { (): any; new(): any; signAndSend: { (arg0: any, arg1: { signer: Signer; }, arg2: any): any; new(): any; }; }; }; }; },
   farmID: number,
-  nodeID: number,
+  nodeId: number,
```

Comment:
why did you change this?

Reply:
No specific reason, I changed it back as it doesn't affect anything.